simplq/package.json

```diff
@@ -27,6 +28,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.4.3",
     "react-spinners": "^0.9.0",
+    "styled": "^1.0.0",
```

Comment:
Is it possible to avoid using styled-components for this task?

Reply:
I have not used styled-components anywhere. This was mistakenly installed.
please review other changes you have asked for.